forge-core/src/main/java/forge/item/PaperCard.java

```diff
@@ -38,7 +37,7 @@
  *
  * @author Forge
  */
-public class PaperCard implements Comparable<IPaperCard>, InventoryItemFromSet, IPaperCard, Serializable {
+public class PaperCard implements Comparable<IPaperCard>, InventoryItemFromSet, IPaperCard {
```

Comment:
Let's keep the random cleanup out of this PR since you aren't even in this file.

Reply:
I initially went into that file and made a modification, but then realized I didn't need it and left the cleanup.

Reply:
to what end is making it unserializable cleanup...?

Reply:
IPaperCard implements it, so it was implementing it twice.